deltacat/compute/resource_estimation/utils.py

```diff
@@ -0,0 +1,415 @@
+import logging
+from typing import Optional, List
+from deltacat import logs
+from deltacat.constants import NULL_SIZE_BYTES
+from deltacat.compute.resource_estimation.parquet import (
+    parquet_column_chunk_size_estimator,
+)
+from deltacat.types.media import ContentEncoding, ContentType
+from deltacat.types.partial_download import PartialParquetParameters
+from deltacat.storage import (
+    ManifestEntry,
+)
+from deltacat.compute.resource_estimation.model import (
+    OperationType,
+    EstimateResourcesParams,
+    ResourceEstimationMethod,
+)
+
+logger = logs.configure_deltacat_logger(logging.getLogger(__name__))
+
+
+def _get_parquet_type_params_if_exist(
+    entry: ManifestEntry,
+) -> Optional[PartialParquetParameters]:
+    if (
+        entry.meta
+        and entry.meta.content_type == ContentType.PARQUET
+        and entry.meta.content_encoding == ContentEncoding.IDENTITY
+        and entry.meta.content_type_parameters
+    ):
+        for type_params in entry.meta.content_type_parameters:
+            if isinstance(type_params, PartialParquetParameters):
+                return type_params
```

Comment:
Will there only ever be one instance of partial parquet parameters that we need to return here?

Reply:
Yes..